post-processor/vsphere-template/post-processor.go

```diff
@@ -37,6 +37,7 @@ type Config struct {
 	SnapshotEnable      bool   `mapstructure:"snapshot_enable"`
 	SnapshotName        string `mapstructure:"snapshot_name"`
 	SnapshotDescription string `mapstructure:"snapshot_description"`
+	ReregisterVM        bool   `mapstructure:"reregister_vm" default:true`
```

Comment:
Perfect -- glad you defaulted to "true" here.

Reply:
Right, I wanted to maintain backwards compatibility, but could not think of a good variable name for "Actually use mark as template" :)

Reply:
naming, the #1 hardest problem...

Reply:
Ah, this reminds me -- you also need to add docs for this new option to https://github.com/hashicorp/packer/blob/master/website/source/docs/post-processors/vsphere-template.html.md

Reply:
ah, quite true.

Reply:
This default tag is invalid and makes the github.com/fatih/structtag crash; here's a fix:
```suggestion
	ReregisterVM        bool   `mapstructure:"reregister_vm" default:"true"`
```

Reply:
This default tag makes the github.com/fatih/structtag crash; here's a fix:
```suggestion
	ReregisterVM        bool   `mapstructure:"reregister_vm" default:"true"`
```